lib/xml/encrypt.js

```diff
@@ -29,8 +29,29 @@ exports.unencrypted = function (xml, callback) {
 exports.encrypted = function (encryptOptions) {
   return function encrypt(xml, callback) {
     xmlenc.encrypt(xml, encryptOptions, function (err, encrypted) {
-      if (err) return callback(err);
-      callback(null, utils.removeWhitespace(encrypted));
+      if (err) {
+        // Attempt to fix errors and retry
+        console.warn(
+            `Failed to encrypt xml. Attempting to fix errors and retrying. Error=${err.message}`
+        );
+        xmlenc.encrypt(
+            xml,
+            {
+              ...encryptOptions,
+              rsa_pub: utils.fixPemFormatting(encryptOptions.rsa_pub),
+              pem: utils.fixPemFormatting(options.encryptionCert),
```

Comment:
`options` doesn't exist (I only noticed this because I ran eslint on this file). This leads me to believe there might not be a test for this case!

Reply:
Fixed. I added the missing tests for encryption